.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @CasperWA @ml-evs @merkys
```

Comment:
I wouldn't mind to be pinged on PRs in this repo.
```suggestion
* @CasperWA @ml-evs @merkys @rartino 
```

Reply:
Great, didn't want to assume but that is why I requested your review :P